cypress/e2e/components/generator.spec.js

```diff
@@ -52,6 +54,8 @@ describe('Request generator tool component', () => {
     }); */
 
     it('loads pdf worker for pdf only companies', () => {
+        skipOn(isOn('production'));
```

Comment:
This is because of https://github.com/datenanfragen/website/issues/918 right? Can you link that issue here in a comment?

Reply:
Nope, `pdfWorker` isn't always loaded currently, it is "just" part of the bundle unnecessarily.

We need to disable because `window.pdfWorker` isn't populated in prod.